Casks/u/universal-control.rb

```diff
@@ -0,0 +1,27 @@
+cask "universal-control" do
```

Comment:
I think we should call this `presonus-universal-control`
```suggestion
cask "presonus-universal-control" do
```